app/main/controllers/template/rtmedia-functions.php

```diff
@@ -1515,10 +1507,20 @@ function rtmedia_pagination_page_link( $page_no = '' ) {
 		$link .= $rtmedia_query->media_query['album_id'] . '/';
 	}
 
-	if ( isset( $rtmedia_query->action_query->media_type ) ) {
+	$allowed_types = array();
+
+	// get all allowed media type
+	foreach ( $rtmedia->allowed_types as $type ) {
+		$name = strtoupper( $type['name'] );
+		$allowed_types[] = constant( 'RTMEDIA_' . $name . '_SLUG' );
+	}
+
+	$media_type = apply_filters( 'rtmedia_media_type', $rtmedia_query->action_query->media_type );
```

Comment:
@BhargavBhandari90 It's not necessary that `media_type` will be always set.
Before accessing any object member make sure it exists.